.github/dependabot.yml

```diff
@@ -3,4 +3,21 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
+    labels:
+      - "dependencies"
+      - "@dev-productivity"
+  - package-ecosystem: "devcontainers"
+    directory: "/"
```

Comment:
Not sure it will work though. Pip is in the Dev Container.

Reply:
I specifically sourced this from another repo with a very similar setup and AFAIR it worked there.